unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/Loader/WearableRetriever.cs

```diff
@@ -74,9 +75,13 @@ void OnFailEventWrapper(Exception e)
                 throw;
             }
         }
+
         private bool CheckGLTFastFeature() =>
             DataStore.i.featureFlags.flags.Get().IsFeatureEnabled(FEATURE_GLTFAST);
 
-        public void Dispose() { loaderAssetHelper?.Unload(); }
+        public void Dispose()
```

Comment:
(Looking for confirmation on this correction, please let me know if this is wrong)

This should be 
```
public void Dispose() =>
     loaderAssetHelper?.Unload();
```